tests/unit/integrations_kafka_consumer.cpp

```diff
@@ -144,7 +144,7 @@ TEST_F(ConsumerTest, BatchInterval) {
   static constexpr auto kMessageCount = 7;
   for (auto sent_messages = 0; sent_messages < kMessageCount; ++sent_messages) {
     cluster.SeedTopic(kTopicName, kMessage);
-    std::this_thread::sleep_for(kBatchInterval * 0.5);
+    std::this_thread::sleep_for(kBatchInterval * 0.6);
```

Comment:
`integrations_kafka_consumer` tests failed a few times before because this time limit exceeded for a few milliseconds.
Even though we could have more precise measurements in this test, we still need to set a hardcoded value that is `<= kBatchInterval`. Currently, it's set to `0.5 * kBatchInterval`, but that seems arbitrary and not very significant, like many of the constants in the test (maybe I'm wrong but that is how it looks to me atm). Increasing this value to 0.6 seems reasonable from my perspective.